source/common/network/listener_impl.cc

```diff
@@ -9,25 +9,62 @@
 #include "common/network/connection_impl.h"
 #include "common/ssl/connection_impl.h"
 
+#include "envoy/network/connection_handler.h"
```

Comment:
can you put this up in "public" includes section (under envoy/common/exception.h)